cfme/containers/provider/__init__.py

```diff
@@ -110,8 +110,8 @@ class ContainersProvider(BaseProvider, Pretty):
         'num_replication_controller',
         'num_pod',
         'num_node',
-        'num_container',
-        'num_image']
+        'num_image_registry',
+        'num_container']
```

Comment:
This should be version picked here if its num_image in 5.6 and num_image_registry in 5.7